lib/fizzbuzz.rb

```diff
@@ -1,9 +1,17 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+  
+	  case 
+	  when input % 3 == 0 && input % 5 != 0
+	  	 "Fizz"
+	  when input % 5 == 0 && input % 3 != 0
+	  	 "Buzz"
```

Comment:
For the Fizz and Buzz portions, do you need to make sure it passes both instances, or would it work so long as one of them checks out?